cypress/integration/gpu-dependant/canvas.js

```diff
@@ -0,0 +1,80 @@
+// Smoke test for Tectonic Explorer
+
+context('Canvas Test', () => {
+  before(function () {
+    cy.visit('/?planetWizard=true')
+    cy.wait(3000)
```

Comment:
There's now a custom Cypress command `cy.waitForSplashscreen()` that does not depend on time (cc @kevinsantos-cc).